runtime/v2/manager_windows.go

```diff
@@ -20,15 +20,11 @@ package v2
 
 import (
 	"github.com/containerd/containerd/platforms"
-	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
-func supportedPlatforms() []ocispec.Platform {
-	return []ocispec.Platform{
-		platforms.DefaultSpec(),
-		{
-			OS:           "linux",
-			Architecture: "amd64",
-		},
+func defaultPlatforms() []string {
+	return []string{
+		platforms.DefaulString(),
```

Comment:
Slight typo; `DefaultString` needs the `t` :)

Reply:
Fixed